cpp/msbuild/ZeroC.Ice.Cpp.targets

```diff
@@ -1,39 +1,29 @@
 <?xml version="1.0" encoding="utf-8"?>
-<!-- Copyright (c) ZeroC, Inc. All rights reserved. -->
+<!-- Copyright (c) ZeroC, Inc. -->
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
-    <!-- Use bin and lib directories that match the used platform toolset -->
     <PropertyGroup>
         <IceConfiguration Condition="'$(UseDebugLibraries)' == 'true'">Debug</IceConfiguration>
         <IceConfiguration Condition="'$(UseDebugLibraries)' != 'true'">Release</IceConfiguration>
+
+        <!-- Set the debugger environment PATH to include the package bin directory for the selected configuration. -->
         <LocalDebuggerEnvironment Condition="'$(LocalDebuggerEnvironment)' == ''">PATH=$(MSBuildThisFileDirectory)bin\$(Platform)\$(IceConfiguration)</LocalDebuggerEnvironment>
-        <IceDllDebugSuffix Condition="'$(IceConfiguration)' == 'Debug'">d</IceDllDebugSuffix>
-        <IceDLLSuffix>38a0$(IceDllDebugSuffix)</IceDLLSuffix>
     </PropertyGroup>
+
     <!-- Import Slice Tools targets file -->
     <Import Project="$(MSBuildThisFileDirectory)..\ZeroC.Ice.Slice.Tools.Cpp.targets" />
 
-    <ItemGroup>
-        <_Ice_DLLs Include="$(MSBuildThisFileDirectory)bin\$(Platform)\$(IceConfiguration)\*$(IceDLLSuffix).dll" Exclude="$(MSBuildThisFileDirectory)bin\$(Platform)\$(IceConfiguration)\icedb38*.dll"/>
-        <_Ice_DLLs Include="$(MSBuildThisFileDirectory)bin\$(Platform)\$(IceConfiguration)\libssl*.dll"/>
-        <_Ice_DLLs Include="$(MSBuildThisFileDirectory)bin\$(Platform)\$(IceConfiguration)\libcrypto*.dll"/>
-        <_Ice_DLLs Include="$(MSBuildThisFileDirectory)bin\$(Platform)\$(IceConfiguration)\bzip2*.dll"/>
-    </ItemGroup>
-
     <ItemDefinitionGroup>
         <ClCompile>
+            <!-- Add the package include directory to the compiler's include path. -->
             <AdditionalIncludeDirectories>$(MSBuildThisFileDirectory)include;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
         </ClCompile>
         <Link>
-            <AdditionalLibraryDirectories>$(MSBuildThisFileDirectory)lib\$(Platform)\Release;$(MSBuildThisFileDirectory)lib\$(Platform)\Debug;%(AdditionalLibraryDirectories)</AdditionalLibraryDirectories>
+            <!-- Add the package library directories for both Debug and Release configurations -->
+            <!-- to the linker library path. -->
+            <AdditionalLibraryDirectories>$(MSBuildThisFileDirectory)lib\$(Platform)\Debug;%(AdditionalLibraryDirectories)</AdditionalLibraryDirectories>
+            <AdditionalLibraryDirectories>$(MSBuildThisFileDirectory)lib\$(Platform)\Release;%(AdditionalLibraryDirectories)</AdditionalLibraryDirectories>
         </Link>
     </ItemDefinitionGroup>
 
-    <Target Name="ValidateSliceCompilerVersion" BeforeTargets="CLCompile">
```

Comment:
That was used when we have multiple zeroc.ice.vXXX packages to ensure you don't mix two version of the package. Now there is a single ZeroC.Ice.Cpp.